README.md

```diff
@@ -19,6 +20,34 @@
 * [Replacing DataPusher with XLoader](#Replacing-DataPusher-with-XLoader)
 
 
+## 0. Getting Started Quickly
+
+These instructions are for a quick CKAN Development environment running on docker:
+
+Clone the Repository and go to the root directory.
+
+	git clone https://github.com/CivityNL/ckan-docker.git
+    cd ckan-docker
+
+Checkout `civity` Branch
+
+    git checkout civity
+
```

Comment:
Also might be easier to make this `git clone --branch civity https://github.com/CivityNL/ckan-docker.git`?

Reply:
I will update this.